src/NServiceBus.Core/Serializers/XML/ReadOnlyMemoryStream.cs

```diff
@@ -1,16 +1,14 @@
-﻿// unset
```

Comment:
If this is going to be used by JSON as well, should it be moved perhaps up a level to the NSB.Core/Serializers directory?